_pages/privacy.md

```diff
@@ -2,10 +2,22 @@
 title: Privacy
 ---
 
-See the following sub-sections:
+<p>The GSA Privacy Office preserves and evolves privacy protections for any individual whose personal information is handled by the agency. In practice, this requires the Office to conduct a thorough, periodic analysis of each of the GSA&rsquo;s programs and information systems.</p>
 
-* [Privacy Impact Assessment (PIA)](pia/)
-* [Personally Identifiable Information (PII)](pii/)
-* [System of Records Notice (SORN)](sorn/)
+<p>As someone interested in shipping a product or service, you&rsquo;ll need to first consider <strong>the agency</strong> at which the product or service will be authorized to operate. If that agency is someone <i>other</i> than GSA, please stop reading this guide and contact the Privacy Office at the authorizing agency. Otherwise, take the following steps:</p>
 
-For help, reach out to [#wg-privacy](https://gsa-tts.slack.com/messages/C4F2HMGBE/).
+<ol>
+  <li>
+    <a href="https://gsa.gov/portal/getMediaData?mediaId=199855">GSA Policy 1878.2A CIO P Conducting Privacy Impact Assessments (PIAs)</a> requires that each of the GSA&rsquo;s information systems, either planned or under development, be subject to an annual privacy threshold analysis. Please <a href="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLScrhrrzgS-ZcOpS37YzLdmVXkwX_3dBefZ1t39oKcgdVepmuQ/viewform">fill out and submit this Google Form</a> to get started.
```

Comment:
Seeing as the "fill out and submit this Google Form" part is the major call to action on this page, I would bold it, and/or put it as a TL;DR at the top.